muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/muikku-material-loader.js

```diff
@@ -530,6 +530,29 @@
           memoFieldElement.addClass('ckeditor-field');
         }
         $(object).replaceWith(memoFieldElement);
+        
+        // #3120 memo word counter (non-richedit)
+        if (data.meta.richedit != true) {
+          var wordCountContainer = $('<div class="word-count-container">')
+            .append($('<span class="word-count-title">').text(getLocaleText('plugin.workspace.memoField.wordCount')))
+            .append('<span class="word-count">');
+          memoFieldElement.after(wordCountContainer);
+          var countMethod = function() {
+            var text = memoFieldElement.val().trim();
+            $(wordCountContainer).find('.word-count').text(text == '' ? 0 : text.split(/\s+/).length);
+          };
+          var timer = undefined;
+          memoFieldElement.on('focus', $.proxy(function() {
+            timer = setInterval($.proxy(function() {
```

Comment:
Can't see how this could ever happen, considering all the related fields have both focus and blur set. No biggie to check, though.